recipes-core/images/openvario-base-image.bb

```diff
@@ -2,8 +2,7 @@ SUMMARY = "A small image just capable of allowing a device to boot."
 
 IMAGE_FEATURES += "splash ssh-server-dropbear package-management"
 DEPENDS += "linux-firmware \
-    git \
-"
+    "
```

Comment:
I think the idea was to allow users to manage their .xcsoar dir with git (if they want to).  That said, I'm not sure this has to be in the base image. I'm fine with removing it. 

@Scumi, if you still need git on openvario, I can make it installable through my https://github.com/kedder/openvario-ppa.

Reply:
But ... who really does that? Even I wouldn't do that - and even if, I would rsync to my main computer and have the git repository only there.
So if we're talking about useful tools for power users, rsync would be one, but not git.

Reply:
I'm fine with this change. It wasn't practical.

Reply:
Would be also ok to remove git and think about a "development image" or a possibility to install the tools afterwards ...

Reply:
Think we can go without git ...